glib/src/quark.rs

```diff
@@ -14,6 +14,11 @@ impl Quark {
         unsafe { from_glib(ffi::g_quark_from_string(s.to_glib_none().0)) }
     }
 
+    #[doc(alias = "g_quark_from_static_string")]
+    pub fn from_static_string(s: &'static str) -> Quark {
+        unsafe { from_glib(ffi::g_quark_from_static_string(s.to_glib_none().0)) }
```

Comment:
This is unsound: the string is not actually static. `to_glib_none().0` has to allocate a new temporary string, append a NUL, pass it to the C function and then frees it.

That's why you directly have to call the FFI function

Reply:
ok